darktheme.css

```diff
@@ -0,0 +1,40 @@
+body {
+    color: var(--secondary-color);
+}
+
+body.dark-theme {
+    color: var(--light-color);
+    background-color: var(--button-floatHover-color);
+}
+
```

Comment:
Appreciation: Great work Jyotsna. Dark theme is really looking dope.  Background color (for the whole page) in dark theme adds a punch to the website.